z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-bootstrap/tasks/main.yaml

```diff
@@ -76,4 +75,33 @@
     volume_size: "{{ bootstrap_flavor_size.stdout_lines[0]}}"
     terminate_volume: True
   when:
-  - disk_type == "scsi"
\ No newline at end of file
+  - disk_type == "scsi"
+  - use_storage_provider == false
+
+- name: 'Create BFV bootstrap volume from backend storage provider'
+  os_volume:
+    state: present
+    name: "{{ os_bootstrap_server_name }}-bfv"
+    image: "rhcos"
+    size: "{{ bootstrap_flavor_size.stdout_lines[0]}}"
+    volume_type: "{{ volume_type_template }}"
+    metadata: "{{ cluster_id_tag }}"
+    timeout: 3600
```

Comment:
why would this hardcode to 3600 instead of using the configuration "default timeout" you added?

Reply:
thanks for your comments, it's already merged.